### DISCUSSION TITLE
Allow per db setting, post-processing command for pdfs

### DISCUSSION THREAD

**Discussion by einhverfr** [ledgersmb/LedgerSMB#5815] [Ideas]
https://sourceforge.net/p/ledger-smb/feature-requests/303/

This would allow signed pdfs among other things

**Comment by ehuelsmann** [comment]
With every per-company configuration now part of the database, this causes very tight integration between in-database state and the underlying system. We need design before we really decide to do this. At least, it's not trivial.

**Comment by ehuelsmann** [comment]
Come to think of it, this has been solved by configuring template processing with processors configured through dependency injection. Marking as closed.
